package.json

```diff
@@ -119,7 +118,7 @@
     "morgan": "1.10.0",
     "path-to-regexp": "6.2.2",
     "polyfill-crypto.getrandomvalues": "1.0.0",
-    "ramda": "patch:ramda@npm:0.28.0#./patches/ramda-npm-0.28.0-6a5fe8f6cc.patch",
+    "ramda": "0.30.0",
```

Comment:
Looks like `0.30.1` is available: https://github.com/ramda/ramda/releases, may be worth going with the latest?

Reply:
I will change this now, thanks 😄